beacon/handler_test.go

```diff
@@ -271,9 +271,15 @@ func (b *BeaconTest) MoveTime(t time.Duration) {
 		fmt.Printf(" - %d increasing time of node %d - %s (pointer %p)- before: %d - current: %d - pointer clock %p\n", time.Now().Unix(), n.index, n.private.Public.Address(), n, before, n.clock.Now().Unix(), n.handler.conf.Clock)
 	}
 	b.time.Advance(t)
-	// give each handlers time to perform their duty
-	time.Sleep(time.Duration(b.n*100) * time.Millisecond)
-	//time.Sleep(100 * time.Millisecond)
+	time.Sleep(getSleepDuration())
+}
+
+func getSleepDuration() time.Duration {
+	if os.Getenv("CIRCLE_CI") != "" {
+		fmt.Printf("\n\n--- Sleeping on CIRCLECI\n\n")
+		return time.Duration(1000) * time.Millisecond
```

Comment:
does it need to be this much? a full second is a lot of time

Reply:
that's the price to pay to stick with ***** dockers :D  If we ever go to more robust machine it'd be fine to lower this but the test were failing because it didn't wait long enough for the nodes to produce the beacon with 500ms , which is already a lot.